ui/app/AppLayouts/Profile/panels/ProfileDescriptionPanel.qml

```diff
@@ -33,7 +33,6 @@ Item {
             label: qsTr("Display name")
             placeholderText: qsTr("Display Name")
             charLimit: Constants.keypair.nameLengthMax
-            validators: Constants.validators.displayName
```

Comment:
Why is this removed? :thinking:

Reply:
Because the same thing (with some other conditions) is set in `ui/app/AppLayouts/Profile/views/MyProfileView.qml`